build/vsts_build.yaml

```diff
@@ -101,13 +101,6 @@ jobs:
         Write-Host "##vso[build.updatebuildnumber]$env:FullVstsBuildNumber"
         Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
 
-  - task: PowerShell@1
```

Comment:
What was this doing?

Reply:
Since GitHub shows the lines above the line where there's a comment, it would help me if you could either do a multi-line comment, or ask these questions on the last line of the block, so I can more easily see what the question was about.

Anyway, it was just setting a build/environment variable that could be used at a later step. I don't know why it wasn't set at the phase, where we already define a few variables, or on the task, where there are also two variables defined. Anyway, I changed things a little, so this task/step is no longer needed.

Reply:
I agree that you shouldn't need to do this this way. Specially now that you don't refer to that variable in YAML.